jujugui/static/gui/src/app/components/credential-add-edit/credential-add-edit.js

```diff
@@ -62,14 +61,15 @@ class CredentialAddEdit extends React.Component {
     @returns {Object} React component for DeploymentCloud.
   */
   _generateChooseCloud() {
+    const cloudFacade = this.props.cloudFacade;
     return (
       <DeploymentCloud
         acl={this.props.acl}
         addNotification={this.props.addNotification}
         cloud={this.state.cloud}
-        controllerIsReady={this.props.controllerIsReady}
+        controllerIsReady={() => true}
```

Comment:
These components weren't able to be updated in this pass, so when I'm able to go back and update them (when I do the DF) then it will be updated and removed.